webapp/channels/src/i18n/en.json

```diff
@@ -29,7 +29,6 @@
   "accessibility.button.bold": "bold",
   "accessibility.button.bulleted_list": "bulleted list",
   "accessibility.button.code": "code",
-  "accessibility.button.dialog": "{dialogName} dialog",
```

Comment:
Dont we need to remove this key from other translation files as well?

Reply:
The PR pipeline doesn't allow making changes in other translation files except `en.json`.